Cargo.lock

```diff
@@ -4342,7 +4342,6 @@ dependencies = [
  "tracing",
  "uniffi",
  "uuid",
- "wasm-bindgen",
```

Comment:
Looks incorrect, should be etire dep

Reply:
Seems like `reqwest` has a dependency to `wasm-bindgen` which is not present in our selection of feature set for request. But it is still locked in `Cargo.lock` file.